src/patternfly/components/Drawer/drawer.scss

```diff
@@ -130,12 +131,12 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   }
 
   @media screen and (min-width: $pf-global--breakpoint--xl) {
-    --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--xl--FlexBasis);
+    --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--m-resizable--FlexBasis, var(--pf-c-drawer__panel--xl--FlexBasis));
```

Comment:
Should this be:

```suggestion
    --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--m-resizable--FlexBasis, var(--pf-c-drawer__panel--xl--FlexBasis, var(--pf-c-drawer__panel--md--FlexBasis)));
```

Reply:
It could be, though I don't think it needs to be. The fallback to `--md` would be needed if `--xl` were an optional var, but since `--pf-c-drawer__panel--xl--FlexBasis` is set by the component, it should always be true. wdyt?